certbot/docs/contributing.rst

```diff
@@ -520,19 +520,21 @@ during the next release.
 Updating the documentation
 ==========================
 
-In order to generate the Sphinx documentation, run the following
-commands:
+Many of the packages in the Certbot repository have documentation in a
```

Comment:
Trying to remove the number of `in` with my "English" : Most of the packages in the Certbot repository have a documentation stored as a ``docs`` directory. This directory is located under the root path of the package.